### PR TITLE
bugfix/19976-jumpy-zoom

### LINKED ISSUE #19976: Jumping of the time period when we zoom in and out or when we shift to the right
#### Expected behaviour
When we pinch in or out it should be work well.

#### Actual behaviour
Jumping of the time period when we zoom in and out or when we shift to the right

#### Live demo with steps to reproduce
https://jsfiddle.net/KrupaliClavier/7vea0Lfg/13/

#### Affected browser(s)
Web view in reactnative

https://github.com/highcharts/highcharts/assets/140229103/a21f52cd-9e99-4df2-97ff-20ecaaae1627

[jumping_to_right.webm](https://github.com/highcharts/highcharts/assets/140229103/7a3229bb-acae-431f-b1c9-7a356a380dab)

### PR DESCRIPTION
**Pull request by DJTechnoo**
Fixed #19976, columns threw off center when using mouse wheel zoom.

### CODE REVIEW

**Review by TorsteinHonsi — CHANGES_REQUESTED** [review]
Thanks! Please add a unit test too.

**Comment by DJTechnoo** [comment]
When all is good, here is a [PR](https://github.com/highcharts/highcharts-utils/pull/77) ready to go for utils

File: samples/unit-tests/chart/mouse-wheel/demo.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        },
+        series: [{
+            type: 'column',
+            data: Array.from(Array(30)).map(() => Math.random() * 10)
+        }]
+    });
+
+    const controller = new TestController(chart);
+
+    controller.mouseWheel(200, 100, -1000, true);
```
Comment (pawelfus) [inline samples/unit-tests/chart/mouse-wheel/demo.js:17]: Nitpicking: unnecessary debug=true

**Review by pawelfus — CHANGES_REQUESTED** [review]

**Review by TorsteinHonsi — APPROVED** [review]
Thanks!

**Pull request merged**